src/test/java/org/jenkinsci/plugins/scriptsecurity/scripts/JcascTest.java

```diff
@@ -0,0 +1,31 @@
+package org.jenkinsci.plugins.scriptsecurity.scripts;
+
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class JcascTest {
```

Comment:
Could you add an export test too please?
https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/PLUGINS.md#model-export-test